components/brave_vpn/browser/connection/ikev2/win/brave_vpn_helper/BUILD.gn

```diff
@@ -17,22 +17,6 @@ config("no-missing-braces") {
   cflags = [ "-Wno-missing-braces" ]
 }
 
-source_set("common") {
```

Comment:
you might as well leave these here for now because moving it doesn't fix the layering violation

Reply:
Fixed